pkg/cidata/cidata.TEMPLATE.d/boot/25-guestagent-base.sh

```diff
@@ -2,6 +2,8 @@
 
 set -eux
 
+: "${LIMA_CIDATA_PREFIX:=/usr/local}"
+
```

Comment:
Why is this needed (also in `40-install-containerd.sh`)?

Lima already provides the default value, so `LIME_CIDATA_PREFIX` should never be empty.

Reply:
Maybe it was not needed, but there were some weird test failures installing to "bin/nerdctl"

Reply:
Will revert